### ISSUE TITLE
Containers are getting exited immediately after start

### ISSUE THREAD

**Issue by starsunny** [kubemove/kubemove#40]
Hi @mynktl 

I have tried deploying this to my K8s cluster but containers are getting killed immediately after starting. and getting the following error in logs.

**standard_init_linux.go:219: exec user process caused: exec format error**

Also while trying to install this from helm, it's giving the below error.

**Error: failed to download "kubemove.io/kubemove"**

**Comment by mynktl** [comment]
Hi @starsunny 
Did you build docker images from master branch?
It might be due to the wrong command line in dockerfile or the different host architecture?

**Comment by starsunny** [comment]
@mynktl Yes I have built images from master branch, and deploy to AWS EKS .

Is there any best practice to follow on host architecture or any other??

**Comment by mynktl** [comment]
yes. host machine and guest container image should have the same architecture. For example, if you have built a docker image on ARM machine and you are running this image on  x86-64 machine then it will throw the above error.

**Comment by starsunny** [comment]
Hi @mynktl Thanks, it worked.

But still I think we need a proper doc here to use this. Like, how shall I start "MoveEngine API with the required configuration to start the data movement or sync" , what will be the config.

How can I start with moving one application/workload from one cluster to another? although I can see my all the containers are running in both SRC and DST cluster.

**Comment by mynktl** [comment]
> Hi @mynktl Thanks, it worked.
> 
> But still I think we need a proper doc here to use this. Like, how shall I start "MoveEngine API with the required configuration to start the data movement or sync" , what will be the config.
> 
> How can I start with moving one application/workload from one cluster to another? although I can see my all the containers are running in both SRC and DST cluster.

Hi @starsunny 
I agree about the documentation part. This repo doesn't have much documentation on usage since running kubemove independently, without plugin, is not usable. You need to install app specific kubemove plugin to sync application data. As of now we have added support for elasticsearch app only. You can find the detailed user guide [here](https://github.com/kubemove/elasticsearch-plugin/blob/master/docs/user-guide.md). This includes configuration for move-pair and move-engine.
